docs/source/schema/unions-interfaces.md

```diff
@@ -197,7 +205,7 @@ For more information, see [Using fragments with unions and interfaces](https://w
 
 ### Resolving an interface field
 
-[As with union types](#resolving-a-union-field), Apollo Server requires interfaces to define a `__resolveType` function to determine which valid object type an interface field contains.
+[As with union types](#resolving-a-union-field), Apollo Server requires interfaces to define a `__resolveType` function to determine which implementing object type an interface field is returning.
```

Comment:
Similar to a previous comment: "interface field" isn't technically a thing, but to reword this correctly reads even worse than in my previous suggestion.

"...to determine which implementing object type a field which returns an interface is returning" 🤢